src/main/java/baekjoon/barkingdog/bfs/Silver2583.kt

```
@@ -0,0 +1,52 @@
+package baekjoon.barkingdog.bfs
+
+fun main() {
```

detekt (1.23.8)

[warning] The function main appears to be too complex based on Cognitive Complexity (complexity: 31). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CognitiveComplexMethod)